### PR TITLE
Add icons to all dropdown menus

### LINKED ISSUE #2811: Add icons to dropdown menus
Some dropdown menus have icons (series on course page), some have not. Check if it makes sense to add icons to more menus:
- course page
- user profile page
- evaluation overview
- (user dropdown)

### PR DESCRIPTION
**Pull request by bmesuere**
This pull request adds icons to additional dropdown menus.

image
image
image
image

The commits were cherry-picked from #2903.

Closes #2811

### CODE REVIEW

**Comment by bmesuere** [comment]
@BTWS2 something is wrong with the code in this pull request:
image

(I also think we don't want icons in the navbar dropdown because of the toggle alignment)

**Comment by BTWS2** [comment]
@bmesuere It's ok for me to revert the changes for the user dropdown.

**Comment by bmesuere** [comment]
I removed the commit that added icons to the navbar menu and fixed the remaining syntax errors in a separate commit.

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -72,25 +72,33 @@
             <ul class="dropdown-menu dropdown-menu-end">
               <% if policy(@course).scoresheet? %>
                 <li>
-                  <%= link_to t("courses.scoresheet.scoresheet"), scoresheet_course_url(@course), class: "dropdown-item" %>
+                  <%= link_to scoresheet_course_url(@course), class: "dropdown-item" do%>
```
Comment (chvp) [inline app/views/courses/show.html.erb:75]: We usually put whitespace after `do`s.

**Review by chvp — CHANGES_REQUESTED** [review]

**Review by niknetniko — APPROVED** [review]
Aside from the already requested changes, the current changes look good to me.

(However, despite the title claiming to do so, this PR doesn't add icons to all dropdown menus: the grading wizard also has a dropdown)
image

**Comment by BTWS2** [comment]
@niknetniko Good point, I would suggest:
[mdi-eye](https://iconify.design/icon-sets/mdi/eye.html) Alle scoreonderdelen zichtbaar maken
[mdi-eye-off](https://iconify.design/icon-sets/mdi/eye-off.html) Alle scoreonderdelen verbergen
[mdi-table-row-plus-after](https://iconify.design/icon-sets/mdi/table-row-plus-after.html) Scoreonderdeel toevoegen aan alle oefeningen

**Review by chvp — APPROVED** [review]

**Review by niknetniko — APPROVED** [review]

**Pull request merged**